### PR TITLE
Song branch

### PR DESCRIPTION
**Pull request by IrynaSemenenko**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 # Log file
 *.log
+!important.log
```
Comment (Hmeliumm) [inline .gitignore:6]: Do you have this file anywhere?

Comment (IrynaSemenenko) [reply .gitignore:6]: No, I don't.

Comment (IrynaSemenenko) [reply .gitignore:6]: Should I remove this line?

Comment (Hmeliumm) [reply .gitignore:6]: Yes, or change it to more realistic file (like report.log)

**Review by Hmeliumm — APPROVED** [review]

**Pull request merged**